app/src/main/java/com/vero/tasky/agenda/presentation/components/DateTimeLineComponent.kt

```diff
@@ -0,0 +1,38 @@
+package com.vero.tasky.agenda.presentation.components
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.vero.tasky.agenda.presentation.util.LocalDateParser
+import com.vero.tasky.ui.theme.Inter400Size16
+import java.time.LocalDateTime
+
+@Composable
+fun DateTimeLineComponent(
```

Comment:
Modifier should be the first optional parameter, after all the mandatory parameters

In this case, you have only 1 optional parameter (Modifier) and the rest are mandatory

So the modifier should be the last parameter here

Else, to call the DateTimeLineComponent you always need to use named parameters